inc/functions.php

```diff
@@ -477,19 +477,8 @@ function preferred_languages_filter_user_locale( $value, $object_id, $meta_key )
  * @return bool Whether to override the .mo file loading.
  */
 function preferred_languages_override_load_textdomain( $override, $domain, $mofile ) {
-	$preferred_locales = preferred_languages_get_list();
-
-	if ( empty( $preferred_locales ) ) {
-		return $override;
-	}
-
 	$current_locale = determine_locale();
```

Comment:
It's unfortunate that the locale is passed to the filter which is why we have to keep it before the filter.